lesson-7/todos/src/components/NewNoteModal.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import Modal from 'react-modal'
+import { connect } from 'react-redux'
+
+import * as shortId from 'shortid'
+import '../styles/NewNoteModal.css'
+import { actionTypes } from '../Reducer'
+
+
+class NewNoteModal extends React.Component {
+  constructor() {
+    super()
```

Comment:
Good practice is to always pass `props` to super in Component; we have seen some weird bugs if you don't

Reply:
Or maybe do directly `state = {}`? One or another.